frontend/src/App.css

```
@@ -62,8 +66,18 @@
+  unicode-range:
+    U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074,
+    U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+}
+
+@font-face {
+  font-family: "Google Sans";
+  font-style: normal;
+  font-weight: 100 900;
+  font-display: swap;
+
+  src: url("/Fonts/Google-Sans/GoogleSans-VariableFont_GRAD,opsz,wght.ttf") format("truetype");
```

Stylelint (17.11.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)